Directory.Build.props

```diff
@@ -6,8 +6,8 @@
     <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
   </PropertyGroup>
   <PropertyGroup>
-    <AwsSdkSnsVersion>3.7.0</AwsSdkSnsVersion>
-    <AwsSdkSqsVersion>3.7.0</AwsSdkSqsVersion>
+    <AwsSdkSnsVersion>3.7.100.14</AwsSdkSnsVersion>
```

Comment:
I think here we are looking for the lowest version that has the binary compatibility we want, so here `3.7.100`

Reply:
LambdaWorker supports batch sends, and that depends on:

```xml
<PackageVersion Include="AWSSDK.SimpleNotificationService" Version="3.7.3" />
<PackageVersion Include="AWSSDK.SQS" Version="3.7.0" />
```

Reply:
It would make sense that batching support arrived in both SDKs in the same version. As Stu says, we should use the earliest possible version that works for the use cases, not whatever happens to be the latest one right now.